python/ray/data/context.py

```diff
@@ -286,6 +288,7 @@ class DataContext:
     min_parallelism: int = DEFAULT_MIN_PARALLELISM
     read_op_min_num_blocks: int = DEFAULT_READ_OP_MIN_NUM_BLOCKS
     enable_tensor_extension_casting: bool = DEFAULT_ENABLE_TENSOR_EXTENSION_CASTING
+    use_arrow_tensor_v2: bool = DEFAULT_USE_ARROW_TENSOR_V2
```

Comment:
can you add a comment in the docstring explaining the differences between v1 and v2, as well as backward compatibility.

Reply:
What docstring are you referring to (attached to what)?

Reply:
Added to the `DEFAULT_USE_ARROW_TENSOR_V2` definition

Reply:
there is a long docstring below `class DataContext` covering all fields. 
I was referring to it. but I think we should break it down to each field definition.